### ISSUE TITLE
Diagnostics is filled with WARNING messages

### ISSUE THREAD

**Issue by victornoel** [jflesch/paperwork#519]
Hi,

It's not a big problem, but it defeat the idea of getting a quick diagnostic because the logs are filled with this message:
`WARNING paperwork.frontend.util.canvas Ignore useless call to redraw()`

I'm on Archlinux with latest gtk3, maybe it is related, not sure :)

**Comment by jflesch** [comment]
Actually, this is the one dialog I didn't design with users in mind but me ;)

One of the thing I often have to ask on bug reports is the output of Paperwork in a terminal.
I created it while starting to port Paperwork to Windows. On Windows, I can't users to start Paperwork from the shell and send me the output. 90% won't know how to do it, and it's much more painful to do on Windows than Linux.
So I've added this dialog that just grab the log output of Paperwork and various stats (I guess it could also be useful on Linux sometimes).

Regarding this warning, a while ago, I realized I call canvas.redraw() far too often for drawer objects that aren't even visible anymore. This reduces performances and in some cases it may even trigger Cairo warnings.
So I added a check to ignore these calls, and a warning because I shouldn't even been doing these calls in a first place.
But I haven't had time to track them all down yet.

So yeah, it's a good thing you opened this ticket. I will remind me I still have a bunch of them to find :)
